contrib/sorter/README.md

```diff
@@ -0,0 +1,23 @@
+#Sorter
+This module provides the SortValues transform, which takes a `PCollection<KV<K, Iterable<KV<K2, V>>>>` and produces a `PCollection<KV<K, Iterable<KV<K2, V>>>>` where, for each primary key `K` the paired `Iterable<KV<K2, V>>` has been sorted by the byte encoding of secondary key (`K2`). It will efficiently and scalably sort the iterables, even if they are large (do not fit in memory).
+
+##Caveats
+* This transform performs value-only sorting; the iterable accompanying each key is sorted, but *there is no relationship between different keys*, as Beam does not support any defined relationship between different elements in a PCollection.
```

Comment:
Realizing I should have done a case insensitive grep for beam in the first place... Have done that now and the only two instances were the ones you found. Fixed